python/openvino_tokenizers/tokenizer_pipeline.py

```diff
@@ -504,14 +514,27 @@ def finalize(self) -> None:
     @classmethod
     def from_hf_json(cls, tokenizer_json: Dict[str, Any]) -> "BPETokenizationStep":
         vocab = [token for token, index in sorted(tokenizer_json["model"]["vocab"].items(), key=lambda x: x[1])]
+        added_tokens = {token["content"]: token["id"] for token in tokenizer_json["added_tokens"] if token["id"]}
+        for token_json in tokenizer_json["added_tokens"]:
+            if token_json["rstrip"]:
+                for new_token in generate_tokens_with_space_symbols(token_json["content"], depth=2):
+                    added_tokens[new_token] = token_json["id"]
+        
+        # TODO: CVS-150387 Implement suffix_indicator.
+        if tokenizer_json["model"]["continuing_subword_prefix"]:
+            raise NotImplementedError("continuing_subword_prefix/suffix_indicator is not implemented yet.")
+        
         return cls(
             unk_token=tokenizer_json["model"]["unk_token"] or "",
             fuse_unk=tokenizer_json["model"]["fuse_unk"] or False,
             suffix_indicator=tokenizer_json["model"]["continuing_subword_prefix"] or "",
             end_suffix=tokenizer_json["model"]["end_of_word_suffix"] or "",
             vocab=vocab,
             merges=tokenizer_json["model"]["merges"],
-            added_tokens={token["id"]: token["content"] for token in tokenizer_json["added_tokens"] if token["id"]},
+            added_tokens=added_tokens,
+            byte_fallback=tokenizer_json["model"]["byte_fallback"],
+            # cache_capacity=max(tokenizer_json["model"].get("cache_capacity", int(len(vocab) * VOCAB_SIZE_CACHE_PROPORTION)), MIN_CACHE_CAPACITY)
+            cache_capacity=20775
```

Comment:
Fix

Reply:
Great catch. Thanks for noticing! I was suspecting i left this there

Reply:
fixed